README.md

```diff
@@ -1,36 +1,232 @@
 # Twilio Breakout Massive SDK for Arduino
 
-Breakout SDK enables developers to exchange data between low-powered devices and IoT cloud services. Breakout makes it easy to adopt low power optimizations provided by the Narrowband network and cellular modem hardware.
+Breakout SDK enables developers to connect their devices to an MQTT broker over Narrowband cellular line.
```

Comment:
Not just MQTT, we provide samples that rely on MQTT but the SDK can be used with any networking protocols.

Reply:
Other protocols are not really tested, so I wanted to leave them out to avoid confusion. I can mention them shortly, and make another section in the detailed guide.

Reply:
I wouldn't limit to MQTT though, both OwlModem's UDP and TCP abstractions have been proven functional by the previous CoAP client and the MQTT client using the TCP abstraction.  I've also recently successfully tested a stand-alone UDP implementation as well.  Labeling it as MQTT feels too limiting and prescriptive to me.